Prace_domowe/Praca_domowa1/Grupa1/Kacper_Grzymkowski/PD1.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Doceniam, że podjęto próby usuwania wartości skrajnych zwłaszcza, że nie było to wymagane w tej pracy domowej. Warto byłoby skomentować dlaczego wybrano akurat takie punkty odcięcia.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>